Toolbox/scripts/dump_pf.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+SCRIPTDIR=$( cd -P -- "$(dirname -- "$(command -v -- "$_")")" && pwd -P )
```

Comment:
Just curios? How is this working and how is this setting the Scriptdir?

Reply:
This came from https://stackoverflow.com/a/819695 and https://stackoverflow.com/a/17977259
I've used things like this for many of my bash scripts in the past, quite handy. In this case it lets you run the script from toolbox installation or directly from SD card.